Dbsetup

Justin8912 opened the pull request.

MBellinghausen commented on server/index.js:
```diff
@@ -0,0 +1,33 @@
+  res.send('hi')
+})
+
+app.listen(PORT, () => {
```
should the app.listen happen before the client connect?

ME2SWE replied:
I believe 'app' is for the node server (express). 'Client' would be for DB connection. So the order shouldn't matter, I think. Correct me if I am wrong ;)

Justin8912 replied:
Howard is right!

MBellinghausen reviewed (commented):
All looks good for a set up! Plenty of stuff you already know about, just console logs and test data, if it runs it looks good, I would double check in the index file if you're getting the first call. Not sure if it would actually do anything, since at the moment the app.listen is just a console log to notify nodemon what port it's on, but if there's a connection, it might happen before the .listen and do some funky stuff dev side.

Justin8912 commented:
@MBellinghausen I have since updated this file and probably shouldn't have created a pull request here lol! In terms of the app.listen I dont believe there needs to be anything special in its callback function right now. My understanding is that app.listen listens to the port and just redirects the specific requests (like app.get('/')) to the specific location where that request is handled. I may end up putting stuff in there later though!

Justin8912 commented:
@MBellinghausen @ME2SWE Hi guys, could y'all review again? I changed the server file a little bit and pulled the database connection and moved it to another file called: connectToDB.js.

So the only files that need to be checked out:
1. ./server/connectToDb.js
2. ./server/index.js

Thanks guys!!
